added release notes for 10.18.0.261 (#356)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-261.md

---
weight: 47
title: 10.18.0.261
layout: redirect
---

{{< c8y-admon-info >}}
These release notes refer to build versions
- Core: 10.18.0.261
- UI: 10.18.0.170
{{< /c8y-admon-info >}}

### Fixes

<table>
<colgroup>
<col style="width: 15%;">
<col style="width:50%;">
<col style="width: 10%;">
<col style="width: 12%;">
<col style="width: 13%;">
</colgroup>
<thead><tr>
<th>
Product area</th>
<th>
Description</th>
<th>
Issue</th>
<th>
Build version</th>
<th>Build comp.</th>
</tr>
</thead><tbody>

<tr>
<td>Web SDK</td>

> <td>The export of the <code>@c8y/client</code> package has been adjusted to also export the types. This was causing issues in certain setups.</td>

Reviewer: nitpicky, but shouldn't the second sentence rather state that the fact that the types were not exported caused the issues? also, what kind of issues?

Author: slightly re-phrased

> <td>The export of the <code>@c8y/client</code> package has been adjusted to also export the types. This was causing issues in certain setups.</td>

Author: ```suggestion
<td>The export of the <code>@c8y/client</code> package was missing the required types which caused issues in certain setups. This has been adjusted.</td>

```
